### PR TITLE
Update to angular 10.0.5

### PR DESCRIPTION
**Pull request by Polatouche**
Hello,
this PR update angular to 10.0.5 using ng update.
Then some minor modifications to app.module.ts needed.
The samples are working fine with that PR.

### CODE REVIEW

File: projects/ngx-img-cropper/package.json [hunk 1]
```diff
@@ -4,5 +4,8 @@
   "peerDependencies": {
     "@angular/common": ">=8.0.0 <10.0.0",
     "@angular/core": ">=8.0.0 <10.0.0"
+  },
+  "dependencies": {
```
Comment (trickeyone) [inline projects/ngx-img-cropper/package.json:8]: I had `ng update` do this to my projects, too. It'd probably be better to move this to `peerDependencies`

**Comment by trickeyone** [comment]
@web-dave Would it be possible to get this merged and a new release for Angular v10 support?

**Comment by web-dave** [comment]
Hi, sorry.
I'll have a look tonight.

**Comment by web-dave** [comment]
new version available

**Comment by web-dave** [comment]
Thx for your effort

**Pull request merged**